wire-ios/Wire-iOS/Resources/ar.lproj/Localizable.strings

```diff
@@ -552,8 +552,8 @@
 "content.system.unverified_self_devices" = "You unverified one of [your devices](%@)";
 "content.system.unverified_other_devices" = "You unverified one of [%1$@’s devices](%2$@)";
 
-"content.system.missing_messages.title" = "لم تستخدم هذا الجهاز لمدة. بعض الرسائل لن تظهر هُنَا.";
-"content.system.missing_messages.subtitle_start" = "في هذه الأثناء،";
+"content.system.missing_messages.title" = "You haven’t used this device for a while. Some messages may not appear here.";
+"content.system.missing_messages.subtitle_start" = "Meanwhile,";
```

Comment:
this was deleted if I remember correctly on release 3.111. I think there's a mixed up

Reply:
ok the fix was not ported to develop 🙃, we'll make a PR for it